README.md

```diff
@@ -8,110 +8,89 @@
+as the unified interfaces across diverse trusted foundations like `RTMR+MRTD+CCEL`
+and `PCR+TPM2`. Learn more details of CCNP design at [CCNP documentation](https://intel.github.io/confidential-cloud-native-primitives/).
 
 ![](/docs/ccnp-architecture-high-level.png)
 
-This project should also be able deployed on [diverse cloud native PaaS frameworks](https://www.redhat.com/en/blog/confidential-computing-use-cases) like confidential cluster, container, `kubevirt` etc.
-An example of landing architecture on confidential cluster is as follows, please
-refer [detail deployment steps](/deployment/README.md)
-
-![](/docs/ccnp-landing-confidential-cluster.png)
-
-Finally, the full trusted chain will be measured into CC report as follows using
-TDX as example:
+Finally, the full trusted chain will be measured into a CC report as follows using
+TDX as an example:
 
 ![](/docs/cc-full-measurement-tdreport.png)
 
 _NOTE:_
 
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
-- Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
-
-
-## 2. Design
+- Please refer to structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
+- The CCNP project collects container level primitives by implementing unified APIs defined in [CC Trusted API](https://github.com/cc-api/cc-trusted-api). The project will be moved to [CC Trusted API](https://github.com/cc-api/cc-trusted-api) in the near future. 
 
-`CCNP` includes several micro-services as BaaS(Backend as a Service) to provides
-cloud native measurement, then exposes `CC trusted API` via cloud native SDK:
 
-- Services are designed to hide the complexity of different TEE platforms and provides
-common interfaces and scalability for cloud-native environment to address the fetching
-the fetching of quote, measurement and event log.
+## 2. Installation
 
-- SDK is provided to simplify the use of the service interface for development,
-it covers communication to the service and parses the results from the services.
-With such SDK, users can perform related actions with one simple API call.
+### 2.1 Configuration for Host and Guest for Host and Guest
 
-- A `CCNP` device plugin is provided as the dependency for services such as Quote
-Server and Measurement Server. It will help with device mount and folder injection
-within the service.
+CCNP collects primitives of confidential cloud native environments running in confidential VMs(CVM), such as Intel® TDX guest. The primitives are not only from the TEE + CVM boot process + CVM OS but also from the environments running workloads, e.g. Kubernetes cluster or Docker containers. Thus, you need to check below configuration for both hosts and guests.
```

Comment:
I would add a space after "confidential VMs" and before the acronym is shown (CVM).  Also, you only have to use the circle R after Intel the first time you define Intel(R) TDX.  Then you can refer to it as Intel TDX.